Update syntax for ifTextExists and textDoesNotExist steps (#3519)
vividus-framework/vividus: docs/modules/plugins/partials/plugin-web-app-steps.adoc

When I scroll element located `id(way_down_button)` into view
----

> === Text steps

Reviewer: missing examples for both steps

When I scroll element located `id(way_down_button)` into view
----

> === Text steps

Reviewer 2: ```suggestion
=== Text validation steps
```

==== Validate the text exists

> Validates that the text presents in the current context.

Reviewer 2: ```suggestion
Validates that the text is presented in the current context.
```

> Context can be defined by the <<_change_context,change context>> steps. If no context is defined, the text will be searched on the whole page.

Reviewer 2: ```suggestion
The context can be set by the <<_change_context,corresponding steps>>. If no context is set, the text will be searched on the whole page.
```

[source,gherkin]
----

> Then text `$text` exists

Reviewer 2: missing step parameter description

Author: fixed

----

> .Check that text 'Contract Us' is present on the page

Reviewer 2: ```suggestion
.Check the text 'Contract Us' is presented on the page
```

[source,gherkin]
----
Given I am on page with URL `https://docs.vividus.dev/`
Then text `Contract Us` exists
----

==== Validate the text does not exists

> Validates that the text does not present in the current context.

Reviewer 2: ```suggestion
Validates that the text is not presented in the current context.
```

> Context can be defined by the <<_change_context,change context>> steps. If no context is defined, the text will be searched on the whole page.

Reviewer 2: ```suggestion
The context can be set by the <<_change_context,corresponding steps>>. If no context is set, the text will be searched on the whole page.
```

[source,gherkin]
----

> Then text `$text` does not exist

Reviewer 2: missing step parameter description

----

> .Check that text 'Deprecated' does not present in the element

Reviewer 2: ```suggestion
.Check the text 'Deprecated' is not presented in the element
```

When I scroll element located `id(way_down_button)` into view
----

> === Text validation steps

Reviewer: missing info on case insensitive test search in particular cases

Reviewer: my bad, there is no case insensitive text search

When I scroll element located `id(way_down_button)` into view
----

=== Text validation steps

==== Validate the text exists

Validates that the text is presented in the current context. Expected text is *case sensitive*.

The context can be set by the <<_change_context,corresponding steps>>. If no context is set, the text will be searched on the whole page.

[source,gherkin]
----
Then text `$text` exists
----

> * `$text` - Expected text

Reviewer 3: ```suggestion
* `$text` - Expected text.
```

> * `$text` - Expected text

Reviewer 2: ```suggestion
* `$text` - Expected text.
```

.Check the text 'Contract Us' is presented on the page
[source,gherkin]
----
Given I am on page with URL `https://docs.vividus.dev/`
Then text `Contract Us` exists
----

==== Validate the text does not exists

Validates that the text is not presented in the current context.

The context can be set by the <<_change_context,corresponding steps>>. If no context is set, the text will be searched on the whole page.

[source,gherkin]
----
Then text `$text` does not exist
----

> * `$text` - Text that should not be exist

Reviewer 3: ```suggestion
* `$text` - Text that should not exist.
```

> * `$text` - Text that should not be exist

Reviewer 2: ```suggestion
* `$text` - Text that should not exist.
```
